Lib/inspect.py

```diff
@@ -447,17 +452,18 @@ def isgenerator(object):
     """Return true if the object is a generator.
 
     Generator objects provide these attributes:
-        __iter__        defined to support iteration over container
-        close           raises a new GeneratorExit exception inside the
-                        generator to terminate the iteration
         gi_code         code object
         gi_frame        frame object or possibly None once the generator has
                         been exhausted
         gi_running      set to 1 when generator is executing, 0 otherwise
-        next            return the next item from the container
-        send            resumes the generator and "sends" a value that becomes
+        gi_yieldfrom    object being iterated by yield from or None
+        
+        __iter__()      defined to support iteration over container
```

Comment:
FYI: I've grouped methods, but I'm not sure if `__iter__` really needs to be here. If `__iter__` is here, then we also need to add `__next__`. Our inspect documentation does not mention the `__next__` method for generators. In my opinion, we should remove this method from the docstring since it does not make things clearer.